win32/devdoc/threadpool_win32.md

```diff
@@ -56,6 +56,24 @@ MOCKABLE_FUNCTION(, THANDLE(THREADPOOL), threadpool_create, EXECUTION_ENGINE_HAN
 
 **SRS_THREADPOOL_WIN32_01_025: [** `threadpool_create` shall obtain the PTP_POOL from the execution engine by calling `execution_engine_win32_get_threadpool`. **]**
 
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_011: [** Otherwise, `threadpool_create` shall switch the state to OPENING. **]**
+
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_026: [** `threadpool_create` shall initialize a thread pool environment by calling `InitializeThreadpoolEnvironment`. **]**
+
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_027: [** `threadpool_create` shall set the thread pool for the environment to the pool obtained from the execution engine by calling `SetThreadpoolCallbackPool`. **]**
+
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_028: [** `threadpool_create` shall create a threadpool cleanup group by calling `CreateThreadpoolCleanupGroup`. **]**
+
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_029: [** `threadpool_create` shall associate the cleanup group with the just created environment by calling `SetThreadpoolCallbackCleanupGroup`. **]**
+
+[Will be moved from threadpool_open.]
+**S_R_S_THREADPOOL_WIN32_01_015: [** `threadpool_create` shall set the state to OPEN. **]**
```

Comment:
>pool_create` shall set the state to OPEN. **]** [](http://example.com/codeflow?start=44&length=47)

I don't think we need tracking of the open state, we are removing open/close.

Reply:
Okay, I just felt like some dependent libraries like Queue Processor may have some state testing UTs which may fail.

Reply:
Removed